eventmesh-dashboard-common/src/main/java/org/apache/eventmesh/dashboard/common/model/RocketmqProperties.java

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package org.apache.eventmesh.dashboard.service.properties;
+package org.apache.eventmesh.dashboard.common.model;
 
```

Comment:
"RocketmqProperties" and "TopicProperties" are two classes that end with "Properties" and are placed in the "model" package. However, their meanings and purposes are completely different. Additionally, "model" and "dto" are two different concepts. I think you should consider renaming the class names and package names.